gensim/models/tfidfmodel.py

```diff
@@ -301,10 +301,12 @@ def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.iden
             Only when `pivot` is not None will pivoted document length normalization be applied.
             Otherwise, regular TfIdf is used.
         slope : float, optional
-            Parameter required by pivoted document length normalization which determines the slope to which
-            the `old normalization` can be tilted. This parameter only works when pivot is defined.
+            It is the parameter required by pivoted document length normalization which determines the slope to which
```

Comment:
This conflict is incorrectly resolved. Please continue with changes in #2102, which should be up-to-date with develop.

Reply:
@piskvorky author (@anotherbugmaster) can't commit to your branch (lack of permissions)

Reply:
Why not add @anotherbugmaster as a committer / collaborator?

Reply:
@piskvorky done
@anotherbugmaster please fix #2102 branch (you have permissions for clone current repository (not your fork), commit & push directly to #2012 branch), sorry for bothering on vacation.